src/App.tsx

```diff
@@ -1,61 +1,34 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo/NewTodo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  function addNewTodo(title: string, userId: number, completed = false) {
+    const maxId = Math.max(...todos.map(item => item.id));
```

Comment:
[CODE QUALITY] - There's a potential bug here. If there are no todos, Math.max() will return -Infinity, which will result in an invalid id for the new todo. You should handle the case when the todos array is empty to ensure a valid id is always generated.

Reply:
Good point, do it! 🔥 
Also u can make a separate `helper` function for it